app/views/categories/dashboard.erb

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="stylesheets/metrics.css">
     <link rel="shortcut icon" type="image/x-icon" href="/images/om-favicon.ico">
   </head>
+  <body background="/images/hippie-young-woman-meditating.jpg">
```

Comment:
If you want Avalon's photo on the Merchant metrics page, I guess I am cool with that. My preference is to the healing crystal image for the metrics page.